code/crates/starknet/host/src/spawn.rs

```diff
@@ -276,6 +276,7 @@ async fn spawn_network_actor(
         rpc_max_size: cfg.consensus.p2p.rpc_max_size.as_u64() as usize,
         pubsub_max_size: cfg.consensus.p2p.pubsub_max_size.as_u64() as usize,
         enable_sync: true,
+        protocol_names: cfg.consensus.p2p.protocol_names.clone(),
```

Comment:
Here we should convert between the type defined in the `config` crate and the one defined in the `network` crate.

It's definitely not great and incurs tons of boilerplate, but we can fix all in go some other time.

Reply:
https://github.com/circlefin/malachite/pull/1179/commits/0bd3b728f806f803d92159519d169974f0442d38